events/views.py

```diff
@@ -36,3 +36,19 @@ def create_event(request):
     else: # If logged in user is accessing the page through a normal GET request
         form = EventForm()
         return render(request, 'create_event.html',{'form':form})
+
+
+def edit_event(request, id):
+    event = Event.objects.get(id=id)
```

Comment:
What if event does not exist? See my implementation of the profiles page in my recent pull request.